### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by Tolik9924**

### CODE REVIEW

File: front-end/src/components/Dashboard/Banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  };
+
+  useEffect(() => {
+    getCountOfTeachers();
+    getCountOfStudents();
+  }, []);
+
+  return (
+    <div>
+      <MouseParallaxContainer
```
Comment (andriydzhrvts) [inline front-end/src/components/Dashboard/Banner/Banner.jsx]: Are you sure that you need to wrap all content in parallax container? You could wrap only parallax child

Comment (Tolik9924) [reply front-end/src/components/Dashboard/Banner/Banner.jsx]: must be resetOnLeave for all childrens

Comment (Tolik9924) [reply front-end/src/components/Dashboard/Banner/Banner.jsx]: + it activates when mouse on banner

File: front-end/src/components/Dashboard/Banner/Banner.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import React, { useEffect, useState } from 'react';
```
Comment (andriydzhrvts) [inline front-end/src/components/Dashboard/Banner/Banner.jsx:1]: Remove images which you don't use ^

Comment (Tolik9924) [reply front-end/src/components/Dashboard/Banner/Banner.jsx:1]: sorry, forgot about that

**Review by 6eluuNiGer — APPROVED** [review]

**Pull request merged**